applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -73,24 +108,44 @@ def AddVariables(self):
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NORMAL)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISTANCE)
 
+        KratosMultiphysics.Logger.PrintInfo("::[PotentialFlowPrimalSolver]:: ", "Variables ADDED")
```

Comment:
Use the name of the class in the logger (it helps in case debugging is required).

Reply:
Ok!